common/djangoapps/student/tests/test_email.py

```diff
@@ -132,6 +137,24 @@ def _assert_activation_email(self, subject, body_fragments):
         for fragment in body_fragments:
             self.assertIn(fragment, msg.body)
 
+    @mock.patch('student.tasks.log')
+    def test_send_email_to_inactive_user(self, mock_log):
+        """
+        To verify that activation email has been sent to
+        an un-activated user(logged-in via social-auth)
```

Comment:
The situation that you are considering, in that case, our platform will not allow to login the inactive user. This situation arises when the inactive user logged in through social-auth, in that case, our platform should have to send the activation email. Our test and code modification are catering this situation.